src/components/Form/SetAclPermsDocContainerForm.jsx

```diff
@@ -48,50 +49,21 @@ const SetAclPermsDocContainerForm = () => {
           append: event.target.setAclPerms.value === 'Give'
         }
       : undefined;
-    const otherPodUsername = event.target.setAclTo.value;
-
-    if (!otherPodUsername) {
-      runNotification('Set permissions failed. Reason: Username not provided.', 5, state, dispatch);
-      setTimeout(() => {
-        clearInputFields();
-      }, 3000);
-      return;
-    }
-
-    if (getPodUrl(otherPodUsername) === podUrl) {
-      runNotification(
-        'Set permissions failed. Reason: Current user Pod cannot change container permissions to itself.',
-        5,
-        state,
-        dispatch
-      );
-      setTimeout(() => {
-        clearInputFields();
-      }, 3000);
-      return;
-    }
-
-    if (permissions === undefined) {
-      runNotification('Set permissions failed. Reason: Permissions not set.', 5, state, dispatch);
-      setTimeout(() => {
-        clearInputFields();
-      }, 3000);
-      return;
-    }
+    const webIdToSetPermsTo = `${permissionState.podUrlToSetPermissionsTo}profile/card#me`;
 
     try {
-      await setDocContainerAclPermission(session, permissions, podUrl, otherPodUsername);
+      await setDocContainerAclPermission(session, permissions, podUrl, webIdToSetPermsTo);
 
       runNotification(
-        `${
-          permissions.read ? 'Give' : 'Revoke'
-        } permission to ${username} for Documents Container.`,
+        `${permissions.read ? 'Give' : 'Revoke'} permission to ${
+          permissionState.podUrlToSetPermissionsTo
+        } for Documents Container.`,
         5,
         state,
         dispatch
       );
     } catch (error) {
-      runNotification('Set permissions failed. Reason: File not found.', 5, state, dispatch);
+      runNotification('FAILED TO SET PERMISSIONS. REASON: File not found.', 5, state, dispatch);
```

Comment:
Same comment as above for casing.

Reply:
Just for matter of importance, like using different <h> tags...  I can switch it back if preferred. I'm not married to the idea.